test/functional/wallet_multiwallet.py

```diff
@@ -27,6 +29,13 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
+    def add_options(self, parser):
+        parser.add_argument(
+            '--data_wallets_dir',
+            default=os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data/wallets/'),
+            help='Test data with wallet directories (default: %(default)s)',
```

Comment:
I'm not 100% fluent in python and haven't tested this but don't you need to use e.g. `%  locals()` at the end to fill the `%(default)s`?

Reply:
This is done by the argparse module. It can be tested with

```sh
$ ./test/functional/wallet_multiwallet.py --help|tail -4
  --data_wallets_dir DATA_WALLETS_DIR
                        Test data with wallet directories (default: /home/marc
                        o/workspace/btc_bitcoin_core/test/functional/data/wall
                        ets/)